ton_client/client/src/crypto/hdkey.rs

```diff
@@ -52,13 +52,13 @@ pub fn hdkey_xprv_from_mnemonic(
 
 #[derive(Serialize, Deserialize, ApiType)]
 pub struct ParamsOfHDKeySecretFromXPrv {
-    /// serialized extended private key
+    /// Serialized extended private key
     pub xprv: String,
 }
 
 #[derive(Serialize, Deserialize, ApiType)]
 pub struct ResultOfHDKeySecretFromXPrv {
-    /// private key
+    /// Private key - unprefixed 0-padded to 64 symbols hex string
```

Comment:
Just `64 symbols hex string` is enough. No need to write about prefix and padding.